definitions/scenarios/upgrade_to_capsule_6_15_z.rb

```diff
@@ -87,6 +87,7 @@ class PostUpgradeChecks < Abstract
     def compose
       add_steps(find_checks(:default))
       add_steps(find_checks(:post_upgrade))
+      add_step(Procedures::Packages::CheckForReboot)
```

Comment:
why isn't the step just tagged with `post_upgrade`?

Reply:
I want it to be the final step so the user sees it.